change/@ni-nimble-components-66b0fc49-9117-4442-8063-c9c8baf59521.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we want to provide any usage guidance about how `frameless` and `full-bleed` should be used alongside `readonly` or `disabled`? For example (not sure if these are 100% true):
1. always use frameless and full bleed if it's readonly
2. don't use frameless and full bleed if it's not readonly
3. only use full bleed if nearby left-aligned controls are also full bleed

If you think it's worth adding, let's add something similar to text field. But I could also see an argument that we either don't have concrete guidance yet or that this is like other invalid combinations that we don't document, so it's not worth documenting this one.

Reply:
I'm not sure. The first item in the list is not true, so we definitely wouldn't want to document that. However, I certainly could see a case for documenting (2). I'm not aware of a use case where we'd want to have a frameless + full-bleed control that is editable (since it doesn't look editable until you hover).

While it isn't a great argument, I'm leaning towards not documenting anything since we don't have concrete guidelines. We also didn't document anything for the frameless + full-bleed text-field (though that's not really a good argument for why we shouldn't document something now).